src/integrators/sdirk_integrators.jl

```diff
@@ -72,15 +67,21 @@ end
 
   cache.ηold = η
   cache.newton_iters = iter
-  u = uprev + z
+  u = u .+ dz
```

Comment:
Yes, that makes sense. But we could e.g. still cache the result of `uprev + ω*zprev + ω*zᵧ` and replace the brackets in lines https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/integrators/sdirk_integrators.jl#L767 and
https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/integrators/sdirk_integrators.jl#L792
with it, couldn't we? We could add `d*z` to this result in every iteration and after the loop to obtain the next `u`. But we should not just increase `u` by some `d*Δz` every time because this would accumulate errors. Is that correct?